ideas/co-design-draft-proposal/index.html

```diff
@@ -0,0 +1,176 @@
+                    <li>Synthesize outcomes into project deliverables</li>
+                    <li>Provide subject matter consultation when needed</li>
+                </ul>
+
+                <h3>Deliverables</h3>
+
+                <ol>
+                    <li>A brief guide on how to do co-design</li>
+                    <li>A series of co-design workshops held with residents in Toronto</li>
+                    <li>Synthesis of the co-design sessions outcomes, resulting in contributions to Sidewalk Labs' Inclusion and Accessibility Principles</li>
+                    <li>Reusable co-design toolkit, consisting of the perspectives, activities and reflection measures</li>
+                </ol>
+
+                <h3>Timeline</h3>
+
+                <p><strong>May</strong>: Planning of initial co-design and hackathon events, creation of short <em>How to Co-Design Guide</em><br />
+                <strong>June to Mid-August</strong>: Several co-design sessions and hackathons<br />
+                <strong>Late August</strong>: Initial synthesis of co-design insights to date; contributions to Sidewalk's draft <em>Inclusion and Accessibility Principles</em> (I&amp;APs) document<br />
+                <strong>September</strong>: Remaining co-design session/hackathons (if needed); second iteration of contributions to I&amp;AP document<br />
+                <strong>October</strong>: Full synthesis of co-design sessions; contributions to version 1.0 of I&amp;APs<br />
+                <br />
+                <strong>Iteratively and ongoing</strong>: Create and share inclusive cities co-design tools</p>
+
+                <img src="proposal-diagram-timeline-04.png" alt="This image displays a horizontal timeline from month of May to November, 2018. Several work streams and project milestones are marked on this timeline. The first work stream 'Planning of co-design and hackathon event' starts sometime in early May and finishes by end of that month. The second work stream 'Co-design sessions and hackathons' begins in June and finishes by end of August. The third work stream 'Remaining co-design session/hackathon (if needed)' starts in the beginning of September and ends in the middle of that month. There is also an ongoing work stream that begins in June and continues to early October called 'Creating and sharing inclusive cities co-design toolkit'. The first milestone is marked sometime in late August, and it is called 'initial synthesis of co-design insights to date; contributions to draft inclusion and accessibility principles document'. The second milestone is scheduled for sometime in late september and is labeled as 'initial synthesis of co-design insights to date; contributions to draft inclusion and accessibility principles document'. The final and last milestone is scheduled sometime in early October and called 'Full synthesis of co-design sessions'; contributions to version 1.0 of inclusion and accessibility principles'." />
```

Comment:
This alt text is a bit on the long side. If we think the content is sufficiently replicated in the text above it, Lisa's advice was to use an aria-labelledby attribute and point to the text above.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute